lib/ReactTags.js

```diff
@@ -223,7 +223,9 @@ class ReactTags extends Component {
     const clipboardData = e.clipboardData || window.clipboardData;
     const string = clipboardData.getData('text');
     const regExp = new RegExp(`[${delimiterChars}]+`);
-    string.split(regExp).forEach((tag) => this.props.handleAddition(tag));
+    string
+      .split(regExp)
+      .forEach((tag) => this.props.handleAddition({ id: tag, text: tag }));
```

Comment:
`dasdf,df,adf,a,fa,df` pasting this will only render `df` as the tag in UI. can you fix this as well in this PR

Reply:
@jpbufe3 This issue is not fixed yet. Can you fix it ?

Reply:
yes it is. please look at the latest version of the code

Reply:
https://github.com/prakhar1989/react-tags/pull/306/files#diff-833c370f0b7e67628628a041d1984df3R228

Reply:
try passing the delimiters as `{[9, 32, 188]}` in example main.js file and paste `dasdf,df,adf,a,fa,df`. Only the last tag gets pasted.

Reply:
I'm sorry. I just tested again and it fails locally, but I'm not quite sure why given the test works. Will take another look.

Reply:
okay. so there's a bug in the `main.js` file for the `handleAddition` function. that being said, there's still a race condition with adding them all at once. I think they need to be de-duplicated in the paste method before sending them across either way

Reply:
this is now fixed. please note that anything that calls `setState` and uses a value from the state must use the callback form in order to avoid out of sync issues.